### ISSUE TITLE
chokidar does not work

### ISSUE THREAD

**Issue by fengyun2** [kimmobrunfeldt/chokidar-cli#36]
- platform

windows 10
- node version

6.7
- chokidar-cli version

1.2.0
- code:

npm scripts:

``` json
    "scripts": {
        "build": "babel src -d dist",
        "watch": "chokidar 'src/**' -c 'npm run build'"
    },
```

run: 

``` bash
npm run watch
```
- Bug

chokidar does not work(but if `globally
npm install -g chokidar-cli` and run `chokidar 'src/**' -c 'npm run build'` can run), I am very confused.

**Comment by nltesown** [comment]
On Windows 7, I'm having a similar problem when running the example cloned from https://github.com/riot/examples/tree/gh-pages/rollup (chokidar-cli is used for a watch script defined in **package.json**).

When I run: `npm run watch` 

I get this status: Watching "'src/**'", "run", "build'" ..

But changes in the `src` folder remain without effect. (The **build** script runs as expected when launched manually.)

Could it be a Windows-related issue?

**Comment by floyd-may** [comment]
Having similar trouble on Windows 10. I've also tried downgrading a few versions and I'm still getting the same behavior. File changes aren't triggering my command if it's configured as a script in package.json.

**Comment by odai-alali** [comment]
I had the same problem on windows, using double quotes solved the problem. it worked running this command for me
```
chokidar "src/app/*.scss" -c "npm run sass:build"
```

**Comment by heinhoang** [comment]
me too, on wins 10.

**Comment by gagan-bansal** [comment]
On Ubuntu its working fine.

**Comment by aureluss** [comment]
me too, on wins 10. "chokidar-cli": "^1.2.0",

**Comment by Nephrited** [comment]
An issue here also. Windows 10, chokidar-cli 1.2.0. It works if run via npx though.

**Comment by rdurgarao** [comment]
My script is running in **Win 8.1 Pro**, I used escape character of double quotations to work around.

chokidar \"./src\" -c \"npm run build\"

**Comment by Samuel0Paul** [comment]
As @rdurgarao pointed out, use double quotes "CMD" instead of single quotes on windows.

In your case, change

> "watch": "chokidar 'src/**' -c 'npm run build'"
 
to

> "watch": "chokidar \"src/**\" -c \"npm run build\""

**Comment by tgfbikes** [comment]
Running Win 10, @rdurgarao 's solution worked for me...

**Comment by Pomax** [comment]
I keep running into this, too, so I filed https://github.com/kimmobrunfeldt/chokidar-cli/pull/57 to hopefully address this with some proper documentation.

**Comment by heikkipora** [comment]
I merged the referenced PR #57 and am closing this issue. Updated the rest of quotes in README.md

**Comment by Pomax** [comment]
Thanks!
